apps/etterlatte-behandling/src/main/kotlin/sak/SakRoutes.kt

```diff
@@ -23,39 +28,66 @@ internal fun Route.sakRoutes(
 ) {
     route("/saker") {
         get {
-            call.respond(Saker(inTransaction { sakService.hentSaker() }))
+            KunSystembruker {
+                call.respond(Saker(inTransaction { sakService.hentSaker() }))
+            }
         }
 
-        get("/{id}") {
+        get("/{$SAKID_CALL_PARAMETER}") {
             val sak = inTransaction {
-                sakService.finnSak(requireNotNull(call.parameters["id"]).toLong())
+                sakService.finnSak(sakId)
             }
             call.respond(sak ?: HttpStatusCode.NotFound)
         }
     }
 
-    get("personer/{$FNR_CALL_PARAMETER}/saker/{type}") {
-        val type: SakType = enumValueOf(requireNotNull(call.parameters["type"]))
-        call.respond(inTransaction { sakService.finnEllerOpprettSak(fnr, type) })
+    post("personer/saker/{type}") {
+        val foedselsnummerDTO = call.receive<FoedselsnummerDTO>()
+        val fnr = foedselsnummerDTO.foedselsnummer
+        withFoedselsnummerBehandling(
```

Comment:
Det hadde vært ganske enkelt å gjenbruke eksisterende `withFoedselsnummer` fra common her også? Så slipper vi å ha to av den.

```
withFoedselsnummer(fnr, sakService) {
    val type: SakType = enumValueOf(requireNotNull(call.parameters["type"]))
    call.respond(inTransaction { sakService.finnEllerOpprettSak(fnr, type) })
}
```
Eneste endringen som trengs er å legge til interfacet `PersonTilgangsSjekk` på SakService og rename metoden som sjekker adressebeskyttelse der.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1124/commits/d04e14d91b017b226f11be2a347e0aca20c62467

Reply:
Det ble ganske smud, bra tips.